extensions/extensions-helm/aissemble-spark-application-chart/values.yaml

```diff
@@ -21,6 +30,9 @@ sparkApp:
       - name: ivy-cache
         persistentVolumeClaim:
           claimName: ivy-cache
+      - name: spark-events
```

Comment:
I: Spark events were previously saved to s3. Changed to save to a PVC

Reply:
Should we include this new property in the README properties section?

Reply:
Good question. There is a section on the shared Ivy cache which is similarly a pvc. I can add a section to the readme for Spark-events